hr_holidays_validity_date/models/hr_holidays.py

```diff
@@ -0,0 +1,51 @@
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, exceptions, _
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from datetime import datetime
+import pytz
+
+
+class HrHolidaysStatus(models.Model):
+    _inherit = "hr.holidays.status"
+
+    date_start = fields.Datetime(string='Start Date')
+    date_end = fields.Datetime(string="Expiry Date")
+    use_validity_dates = fields.Boolean()
+
+    _sql_constraints = [
+        ('date_check',
+         "CHECK ( (use_validity_dates=FALSE) OR (date_start <= date_end))",
+         "The start date must be anterior to the end date."),
```

Comment:
Careful with the translation!

Reply:
I have seen this sentence being used by odoo or in other OCA modules but I can change it for "The start date must be greater than the end date."